packages/app-store/_utils/oauth/OAuthManager.test.ts

```diff
@@ -828,6 +828,38 @@ describe("Credential Sync Disabled", () => {
       );
     });
   });
+
+  // Credentials might have no userId attached in production. They could instead have teamId
+  test("OAuthManager without resourceOwner.id is okay", async () => {
```

Comment:
Nice one - do we test the flow anywhere with a teamId? I can't seem to find it when lookign in this file